frontend/src/App.js

```diff
@@ -21,7 +21,7 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="questions" element={<Questions />} />
         <Route path="questions/:page" element={<Questions />} />
-        <Route path="match" element={<Match />} />
+        <Route path="matching" element={<Match />} />
```

Comment:
is there a reason this is changed?

Reply:
/match was using to be a path prefix for deployment and route path, it would be better to seperate these two.